Sources/MongoSwift/ReadConcern.swift

```diff
@@ -2,53 +2,69 @@ import CLibMongoC
 
 /// A struct to represent a MongoDB read concern.
 public struct ReadConcern: Codable {
+    /// Local ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-local/.
+    public static let local = ReadConcern(.local)
+
+    /// Available ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-available/.
+    public static let available = ReadConcern(.available)
+
+    /// Linearizable ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-majority/.
+    public static let linearizable = ReadConcern(.linearizable)
+
+    /// Majority ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-linearizable/.
+    public static let majority = ReadConcern(.majority)
+
+    /// Snapshot ReadConcern, see https://docs.mongodb.com/master/reference/read-concern-snapshot/.
+    public static let snapshot = ReadConcern(.snapshot)
+
+    /// Server default ReadConcern.
+    public static let serverDefault = ReadConcern(nil)
+
+    public static func other(_ level: Level) -> ReadConcern {
+        ReadConcern(level)
+    }
+
+    public static func other(_ level: String) -> ReadConcern {
+        ReadConcern(Level(level))
+    }
+
     /// An enumeration of possible ReadConcern levels.
-    public enum Level: RawRepresentable, Codable, Equatable {
+    public struct Level: Codable, Equatable, LosslessStringConvertible {
```

Comment:
do we need a public level type, or even a separate level type at that? it seems to me like we could just have `ReadConcern` store a string itself?